data_utils/custom_cb.py

```diff
@@ -49,6 +54,10 @@ def finalize(self, status):
         # Optional. Any code that needs to be run after training
         # finishes goes here
         print(f"Finalizing with status: {status}, last metrics: {self.last_metrics}")
-        if self.log_url:
+        if self.response_url:
             requests.post(self.log_url, json={'status': status, 'task_id': self.task_id, 'is_finished': True,
```

Comment:
please fix this url use response_url

Reply:
Thank you for mentioning this point,
Done